README.md

```diff
@@ -17,7 +17,13 @@ You must make 3 changes to your project to get started:
 
 Check out this [demo project](https://github.com/beyonk-adventures/now-sapper-demo) that uses this builder. It can be used as a template, or a way to verify correct usage of the following instructions.
 
-##### 1. Configure `vercel-sapper` as builder in `vercel.json`
+##### 1. Install `vercel-sapper` as part of `devDependencies`
+
+```bash
+ npm install -D vercel-sapper
```

Comment:
You don't need to install the builder as a dev dependency. If you do so, Vercel will simply ignore it and install its own copy from npm.

Reply:
Thanks. Did not know that. I have updated it.